### DISCUSSION TITLE
seeking output of coefficients Cnlm from SOAP descriptors

### DISCUSSION THREAD

**Discussion by taccchen** [SINGROUP/dscribe#120] [General]
Dear DScribe developers, 

   I am a new user of DScribe, and would like to print out the coefficients of Cnlm from the Smooth Overlap of Atomic Positions (SOAP) descriptors. I tried to set average="off", but the output feature vectors do not look like the coefficients of Cnlm. 

  I've attached a PDF for a better explanation of my question. 

  Thanks in advance for your kind help.

Cheers,

Hanning 
[Coefficients_from_SOAP.pdf](https://github.com/SINGROUP/dscribe/files/12210383/Coefficients_from_SOAP.pdf)

**Comment by lauri-codes** [comment]
Hi @taccchen,

The Cnlm coefficients are not directly exposed, as it is an intermediate result while obtaining Pnlm. If you want to get your hands on the values, you will need to modify the source code somewhat.

**Comment by taccchen** [comment]
Dear Lauri,

  Thanks for your clarification.

   I will read the source code.

Cheers,

Hanning


From: Lauri Himanen ***@***.***>
Reply-To: SINGROUP/dscribe ***@***.***>
Date: Wednesday, August 2, 2023 at 11:30 AM
To: SINGROUP/dscribe ***@***.***>
Cc: Hanning Chen ***@***.***>, Mention ***@***.***>
Subject: Re: [SINGROUP/dscribe] seeking output of coefficients Cnlm from SOAP descriptors (Discussion #120)


Hi @taccchen<https://github.com/taccchen>,

The Cnlm coefficients are not directly exposed, as it is an intermediate result while obtaining Pnlm. If you want to get your hands on the values, you will need to modify the source code somewhat.
